tests/test_tezos_interop.re

```diff
@@ -216,9 +245,19 @@ describe("pack", ({test, _}) => {
   );
   test(
     "key(\"edpkvDqjL7aXdsXSiK5ChCMAfqaqmCFWCv7DaT3dK1egJt136WBiT6\")",
-    key(Ed25519(Address.genesis_address)),
+    key(Ed25519(genesis_address)),
     "050a0000002100d00725159de904a28aaed9adb2320f95bd2117959e41c1c2377ac11045d18bd7",
   );
+  test(
+    "key_hash(\"tz1LzCSmZHG3jDvqxA8SG8WqbrJ9wz5eUCLC\")",
+    key_hash(Key_hash.of_key(Ed25519(genesis_address))),
+    "050a00000015000ec89608700c0414159d93552ef9361cea96da13",
+  );
+  test(
+    "address(\"tz1LzCSmZHG3jDvqxA8SG8WqbrJ9wz5eUCLC\")",
+    address(Implicit(Key_hash.of_key(Ed25519(genesis_address)))),
+    "050a0000001600000ec89608700c0414159d93552ef9361cea96da13",
```

Comment:
Might want to include a comment referencing https://ide.ligolang.org/p/4fLtnmyjluBgNoZ-TvBu7A

Reply:
#40 opened an issue on this, I think it's important

Reply:
@anchpop Are you asking how the value expected key_hash was arrived at?

Reply:
It was arrived at using the ligo code @EduardoRFS wrote in the "manual testing" section, but I thought it was unclear in the code why it was being tested against that specific value

Reply:
Closing this, if you want to discuss please use the issue